doc/api/fs.md

```diff
@@ -430,8 +430,8 @@ included in the iteration results.
 added: v10.10.0
 -->
 
-A representation of a directory entry, as returned by reading from an
-[`fs.Dir`][].
+A representation of a directory entry, or a mapping of a filename to an
+inode, as returned by reading from an [`fs.Dir`][].
```

Comment:
Let's try to be a bit more specific here. Someone who doesn't know what a directory entry is, is also unlikely to know what an inode is – let alone the fact that inodes don't exist in certain file systems that we support, like NTFS.

Could you say something in the sense of "A representation of an entry of a file system directory. Examples for directory entries include files and subfolders in a folder that is being read from. Data included in [`fs.Dirent`][] objects include the filename in the directory it belongs to, and also possibly certain metadata about the type of the directory entry, but not the actual file content."

(I'm not a native English speaker, so you could probably make it sound a bit better :)